lib/tapioca/compilers/dsl/active_record_associations.rb

```diff
@@ -153,6 +154,10 @@ def populate_associations(mod, constant)
             add_error(<<~MSG)
               Cannot generate association `#{reflection.name}` on `#{constant}` since the source of the through association is missing.
             MSG
+          rescue MissingConstantError
+            add_error(<<~MSG)
```

Comment:
We probably want to `strip` the error message, since as it is now, it has a dangling `\n` at the end due to the heredoc usage:
```suggestion
            add_error(<<~MSG.strip)
```

We probably want to do the same this for the other error message too, btw.